app/main.py

```diff
@@ -2,0 +1,41 @@
+from __future__ import annotations
+
+
+# Fixed class
+class Animal:
+    alive: list = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def is_alive(self) -> bool:
+        return self.health > 0
+
+    @staticmethod
+    def remove_dead(animal: Herbivore) -> None:
```

Comment:
The `animal` parameter in the `remove_dead` method should be annotated as `Animal`, not `Herbivore`. This is because the method should be able to accept any `Animal` instance, not just `Herbivore` instances. This is a code style issue.